includes/objectcache/MemcachedClient.php

```diff
@@ -1442,6 +1442,6 @@ function _decode_keys( $result, $rev_keys ) {
 
 class MemCachedClientforWiki extends MWMemcached {
 	function _debugprint( $text ) {
-		wfDebug( "memcached: $text" );
+		// Wikia change useless debug removed
```

Comment:
This debug function is quite useful when, well, debugging... 

```
> User::newFromName('Macbre')->load()
memcached: get(wikicities:user:name:Macbre)
memcached: MemCache: sock i:0; got wikicities:user:name:Macbre
memcached: get(wikia:user:id:119245)
memcached: MemCache: sock i:0; got wikia:user:id:119245
memcached: get(wikicities:user_touched:v1:119245)
memcached: MemCache: sock i:0; got wikicities:user_touched:v1:119245
User: got user 119245 from cache
```

Reply:
I wonder how we could make it faster

Reply:
Let's keep `wfDebug` for now and create a ticket for the above performance issue.